podcasts/TranscriptSearchAccessoryView.swift

```diff
@@ -0,0 +1,166 @@
+import UIKit
+import PocketCastsUtils
+
+protocol TranscriptSearchAccessoryViewDelegate: AnyObject {
+    // When "Done" is tapped on this view
+    func doneTapped()
+
+    // When "Search" is tapped on the keyboard
+    func searchButtonTapped()
+}
+
+class TranscriptSearchAccessoryView: UIInputView {
+    weak var delegate: TranscriptSearchAccessoryViewDelegate?
+
+    lazy var textField: CustomTextField = {
+        let textField = CustomTextField()
+        textField.translatesAutoresizingMaskIntoConstraints = false
+        configureTextField(textField)
+        return textField
+    }()
+
+    private lazy var mainStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .vertical
+        stackView.layoutMargins = UIEdgeInsets(top: 8, left: 0, bottom: 8, right: 0)
+        stackView.isLayoutMarginsRelativeArrangement = true
+        return stackView
+    }()
+
+    private lazy var innerStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.axis = .horizontal
+        return stackView
+    }()
+
+    private lazy var doneButton: UIButton = createButton(
+        title: L10n.done,
+        action: #selector(done),
+        titleColor: .label,
+        highlightedTitleColor: .systemGray,
+        contentInsets: .init(top: 8, leading: 16, bottom: 8, trailing: 16)
+    )
+
+    private lazy var downButton: UIButton = createSymbolButton(
+        imageName: "chevron.down",
+        action: #selector(done),
```

Comment:
I know that the search logic is not done yet, but shouldn't this trigger a different delegate method, like a Next/Previous call?

Reply:
Yes, I'll do it on the next PR! Just did that to test that the buttons are actually working as intended.